plutus-core/testlib/UntypedPlutusCore/Test/DeBruijn/Bad.hs

```diff
@@ -83,8 +83,7 @@ deepOutMix1_0 n = timesA n lamAbs1 $
 -- Both branches are evaluated *before* the predicate,
 -- so it is clear that this should fail in every case.
 iteStrict0 :: Term DeBruijn DefaultUni DefaultFun ()
-iteStrict0 = mkIterAppNoAnn
-             (Force () (Builtin () IfThenElse))
+iteStrict0 = Force () (Builtin () IfThenElse) @@
```

Comment:
:o when did we get this?

Reply:
When u were on vacation I secretly introduced it :see_no_evil: